05-JWT-Basics/starter/middleware/auth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken')
+const { UnauthenticatedError } = require('../../final/errors')

```

Comment:
You shouldn't request the `UnauthenticatedError` from the `final` folder, you should get it from your own version (`starter`).

```suggestion
const { UnauthenticatedError } = require('../errors')
```